valusets/ValueSet-England-LocationType.xml

```diff
@@ -0,0 +1,121 @@
+<ValueSet xmlns="http://hl7.org/fhir">
+    <id value="England-LocationType" />
```

Comment:
How is this valueset, England-LocationType related to England Organization?

Reply:
Pushed change. File removed